### ISSUE TITLE
fine-tune with pre-trained model

### ISSUE THREAD

**Issue by kenyeung128** [zhang0jhon/AttentionOCR#37]
might i ask if it's possible to finetune with pre-trained model with extra data

1. if i can add new characters in the file below and then train with existing model?https://github.com/zhang0jhon/AttentionOCR/blob/master/label_dict/icdar_labels.txt

2. if using https://github.com/Belval/TextRecognitionDataGenerator for synthesize text, what about be the masks, bboxes and points for the data?

thanks.

**Comment by zhang0jhon** [comment]
Of course you can add new characters, you just need to exclude the final softmax layer parameters with pre-trained model. As for synthesize text, the mask is the ground-truth sequence length including "EOS"(seq_len+1) which is used to compute sequence classification loss in dynamic sequence length. Bboxes should be the same as points: [[0,0],[width-1,0],[width-1,height-1],[0,height-1]] including the whole synthesize text.

**Comment by johnbird7** [comment]
@zhang0jhon would u please tell me how to prepare dataset of my own with ground truth and finetune on text_recognition_5435.pb checkpoint? :( plz...
